frontend/src/components/survey-item/survey-item.test.tsx

```diff
@@ -59,15 +60,19 @@ describe('SurveyItem', () => {
     ).toBeTruthy();
   });
 
-  it('should have random image in cover', () => {
+  it('should have a given image in cover', () => {
+    const rendered = render(<SurveyItem {...mockSurveyDefault} cover={'test.png'} />);
+    expect(rendered.getByTestId('cover-image')).toHaveStyle(
+      `background-image: url('test.png')`
+    )
+  });
+
+  it('should have a random image in cover', () => {
     const rendered = render(
-      <SurveyItem
-        title="Survey name"
-        description="Available your instincts now"
-        numberQuestions={10}
-      />
+      <SurveyItem {...mockSurveyDefault} cover={''} />
     );
-    expect(rendered.getByTestId('random-image')).toBeTruthy();
+    const coverImageUrl = getComputedStyle(rendered.getByTestId('cover-image')).backgroundImage;
+    expect(images.includes(coverImageUrl.replace(/^url\(['"]?/, '').replace(/['"]?\)$/, ''))).toBeTruthy();
```

Comment:
Can you think of some way to improve the readability of these 2 `replace` methods?
(just a suggestion, not a blocker)

Reply:
Sure!

Reply:
@edimossilva please, check if it's better now.